staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/cbor.go

```diff
@@ -100,15 +100,20 @@ func (s *serializer) Identifier() runtime.Identifier {
 }
 
 func (s *serializer) Encode(obj runtime.Object, w io.Writer) error {
```

Comment:
+Godoc here?  Anything worth saying abou handling of unstructured vs other runtime.Objects?

Reply:
Added some text here and mentioned unstructured.